inception-ui-kb/src/main/java/de/tudarmstadt/ukp/inception/ui/kb/feature/FactLinkingConstants.java

```diff
@@ -19,12 +19,14 @@
 
 public final class FactLinkingConstants
 {
-    public static final String SUBJECT_LINK = "webanno.custom.FactSubjectLink";
-    public static final String OBJECT_LINK = "webanno.custom.FactObjectLink";
-    public static final String QUALIFIER_LINK = "webanno.custom.FactQualifierLink";
+    public static final String SUBJECT_LINK = "de.tudarmstadt.ukp.inception.FactSubjectLink";
+    public static final String OBJECT_LINK = "de.tudarmstadt.ukp.inceptionFactObjectLink";
+    public static final String QUALIFIER_LINK = "de.tudarmstadt.ukp.inception.FactQualifierLink";
 
     public static final String SUBJECT_ROLE = "subject";
     public static final String OBJECT_ROLE = "object";
 
-    public static final String FACT_LAYER = "webanno.custom.Fact";
+    public static final String FACT_LAYER = "de.tudarmstadt.ukp.inception.Fact";
+
+    public static final String LINKED_LAYER_FEATURE = "identifier";
```

Comment:
Please add a comment that `identifier` here is a feature of the `NamedEntity` layer.